### PR TITLE
Add security & compliance scans

### LINKED ISSUE #29: Implement security & compliance scans
- [x] JS dependency audit
- [x] python dependency audit
- [x] static security scanner
- [x] dynamic security scanner
- [x] a11y scanner

### PR DESCRIPTION
**Pull request by rahearn**
As part of every push, and on a daily schedule:

* pip audit
* npm audit
* bandit python static scan
* OWASP Zap dynamic scan
* pa11y-ci accessibility scan

Closes #29

### CODE REVIEW

File: app/broadcast_areas/repo.py [hunk 1]
```diff
@@ -127,13 +127,10 @@ def get_libraries(self):
         return sorted(libraries)
 
     def get_areas(self, area_ids):
-        q = """
-        SELECT id, name, count_of_phones, broadcast_area_library_id
-        FROM broadcast_areas
-        WHERE id IN ({})
-        """.format(",".join("?" * len(area_ids)))
+        q = "SELECT id, name, count_of_phones, broadcast_area_library_id FROM broadcast_areas"
+        where = "WHERE id IN ({})".format(",".join("?" * len(area_ids)))
```
Comment (rahearn) [inline app/broadcast_areas/repo.py:131]: [python newbie question] I don't love this. Do we really need to include a `?` for each id that we want to insert into the statement?

File: app/config.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ class Config(object):
 
 class Development(Config):
     ADMIN_BASE_URL = 'http://localhost:6012'
-    BASIC_AUTH_FORCE = True
+    BASIC_AUTH_FORCE = False
```
Comment (rahearn) [inline app/config.py:105]: [question] Any reason we should be enforcing basic auth locally?

File: app/broadcast_areas/repo.py (hunk 1 above)
Comment (jimmoffet) [reply app/broadcast_areas/repo.py:131]: This seems to be the [preferred param substitution method](https://docs.python.org/2/library/sqlite3.html#cursor-objects) for lists for the sqlite3 driver.

Comment (rahearn) [reply app/broadcast_areas/repo.py:131]: Yeah, that's what I gathered. It makes sense but I prefer other libraries where you can have `... WHERE id IN (?)` and then the library can intelligently sub in a list of ids into the single `?` so I wanted to be sure that wan't possible here either.

Comment (jimmoffet) [reply app/broadcast_areas/repo.py:131]: sqlite3 has an executeMany which you could potentially sub for execute, but you'd need to refactor a bit to use it

Comment (rahearn) [reply app/broadcast_areas/repo.py:131]: awesome, thank you. Not going to update this code, but I'll keep that in mind for future use.

**Pull request merged**